tests/client/mod.rs

```diff
@@ -206,7 +205,59 @@ async fn full() {
 }
 
 #[test]
-fn test_config_init() {
+fn run() {
+    let example_dir = format!("{WORKSPACE_DIR}/tests/client/data/wasm_example");
+
+    cmd!(
+        fail: // when not specifying a wasm module
+        "enarx run",
+        text:
+        "error: The following required arguments were not provided:
+  <MODULE>
+
+Usage: enarx run <MODULE>
+
+For more information try '--help'"
+    );
+
+    // TODO: Get a better test.
+    // Ideally the wasm example would print "Hello World" when no arguments are passed,
+    // and otherwise read the passed-in arguments to say "Hello $FOO" for each argument.
+    // This would let us better demonstrate configuration.
+    cmd!(
+        succeed: // when no configuration is given
+        "enarx run {example_dir}/main.wasm",
+        text:
+        "Hello, world!"
+    );
+
+    cmd!(
+        succeed: // when configuring via file
+        "enarx run --wasmcfgfile {example_dir}/Enarx.toml {example_dir}/main.wasm",
+        text:
+        "Hello, world!"
+    );
+
+    cmd!(
+        succeed: // when configuring via CLI
+        r#"enarx run
+        --with-files '[{{kind = "stdin"}}, {{kind = "stderr"}}, {{kind = "stderr"}}]'
+        {example_dir}/main.wasm"#,
+        text:
+        "Hello, world!"
+    );
+
+    cmd!(
+        succeed: // when configuring via file and overriding via CLI
+        r#"enarx run
+        --wasmcfgfile {example_dir}/Enarx.toml
+        --with-files '[{{kind = "stdin"}}, {{kind = "stdin"}}, {{kind = "stdin"}}]'
```

Comment:
How about we don't do that yet and wait until file management is reworked in #2245, which would allow this to be implemented in much cleaner manner?

Reply:
> I'm really not a fan of passing TOML on command-line

We've already determined that we need to pass config arguments on the command line somehow, and this seems like the least bad option. The alternatives are either using JSON, which can't be copy-pasted to/from the config file and is even more verbose, or having some custom syntax which has the same problem as well as the downside of forcing the user to learn it.